src/mem/DRAMInterface.py

```diff
@@ -1414,3 +1414,97 @@ class LPDDR5_6400_1x16_8B_BL32(LPDDR5_6400_1x16_BG_BL32):
     tCCD_L = "0ns"
     tRRD_L = "0ns"
     tWTR_L = "0ns"
+
+class LLM2(DRAMInterface):
+
+    beats_per_clock = 64
+
+    # 500 MHz for 2GB/s per Die
+    tCK = '2ns'
+
+    # Each channel connecting to a single optical IO
+    device_bus_width = 1
+
+    # Using close-page  policy to  have deterministic
+    # memory access latency
+    page_policy = 'close'
+
+    # Total number of pins are 1024
+    # There are 1 pin/channel
+    # Atom size =  32Byte , BL= 64/1 (Bytes)
+    burst_length = 512
```

Comment:
This confuses me a bit.  This is the number of beats needed for 64 bytes of data? Since it's fully serialized, you need 64*8=512 beats?

Putting this in the comment would be helpful!